main/app/contextualLayer/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright: (C) 2015 WYSIWYD Consortium
+# Authors: Clement Moulin-Frier
+# CopyPolicy: Released under the terms of the GNU GPL v2.0.
+
+# add_subdirectory(goalManager)
```

Comment:
In this case, please delete all files belonging to `goalManager` in this pull request (you can change them to another branch so they don't get lost.